modules/sso.php

```diff
@@ -475,6 +475,7 @@ function login_form() {
 							);
 						?>
 					</h2>
+					<?php do_action( 'jetpack_sso_login_form' ); ?>
```

Comment:
In addition to what @roccotripaldi mentioned already, would it make sense to make the hook name more specific, so it's clear that it's called before the SSO button?

Also, it's worth mentioning that this hook is currently inside the user info `<div />`, so the hook name might make sense to indicate that. Or do we want to move the hook outside the div, so it would be between the user info and the SSO button?

Reply:
In addition to the above, I would recommend adding a docblock to the hook, so it can be parsed and displayed [in the Jetpack Codex](https://developer.jetpack.com/). Here is a good example:

https://github.com/Automattic/jetpack/blob/8d47d97aae4fb6174c3254f808b7e9fc45d1e40e/modules/sso.php#L334